src/itemData/index.tsx

```diff
@@ -8,6 +8,9 @@ interface ItemDataProps {
   readonly mainInfo: string
   readonly className?: Classcat.Class
   readonly mainTitle?: string
+  readonly isMainTitleInfo?: boolean
+  readonly mainTitleInfoIcon?: string
+  readonly onClickInfo?: (event: React.MouseEvent<HTMLElement>) => void
```

Comment:
React.MouseEvent on touch?

Reply:
I'm also asking myself about onClick, onTouchStart, onTouchEnd, onMouseDown, onMouseUp, onBlur and onFocus like in the Button component.